sdk/utils/futures.ts

```diff
@@ -406,8 +408,9 @@ export const POTENTIAL_TRADE_STATUS_TO_MESSAGE: { [key: string]: string } = {
 	INSUFFICIENT_FREE_MARGIN: `You don't have enough sUSD for this trade`,
 };
 
-export const getPythNetworkUrl = (networkId: NetworkId) => {
-	return networkId === 420 ? 'https://xc-testnet.pyth.network' : 'https://xc-mainnet.pyth.network';
+export const getPythNetworkUrl = (networkId: NetworkId, currentServer: PriceServer = 'KWENTA') => {
```

Comment:
I think we could rename `currentServer` to just `server` as current sort of implies what's being used now